crypto/context.c

```diff
@@ -343,7 +343,7 @@ static int context_deinit(OSSL_LIB_CTX *ctx)
 }
 
 #ifndef FIPS_MODULE
-/* The default default context */
+/* The default context */
```

Comment:
Hmm... this is actually incorrect - the default default is meant to be there. Because there is thread default libcontext and this process default context which is used by default on a thread but can be overridden with a different library context created by the application.

Reply:
Reverted